api/admin/controller.py

```diff
@@ -1008,38 +1008,36 @@ def collections(self):
             "fields": [
                 { "key": "external_account_id", "label": _("Library ID") },
                 { "key": "website_id", "label": _("Website ID") },
-                { "key": "username", "label": _("Client Key") },
-                { "key": "password", "label": _("Client Secret") },
+                { "key": ExternalIntegration.USERNAME, "label": _("Client Key") },
+                { "key": ExternalIntegration.PASSWORD, "label": _("Client Secret") },
             ],
         })
 
         protocols.append({
             "name": ExternalIntegration.BIBLIOTHECA,
             "fields": [
-                { "key": "username", "label": _("Account ID") },
-                { "key": "password", "label": _("Account Key") },
+                { "key": ExternalIntegration.USERNAME, "label": _("Account ID") },
+                { "key": ExternalIntegration.PASSWORD, "label": _("Account Key") },
                 { "key": "external_account_id", "label": _("Library ID") },
             ],
         })
 
         protocols.append({
             "name": ExternalIntegration.AXIS_360,
             "fields": [
-                { "key": "username", "label": _("Username") },
-                { "key": "password", "label": _("Password") },
+                { "key": ExternalIntegration.USERNAME, "label": _("Username") },
+                { "key": ExternalIntegration.PASSWORD, "label": _("Password") },
                 { "key": "external_account_id", "label": _("Library ID") },
-                { "key": "url", "label": _("Server") },
+                { "key": ExternalIntegration.URL, "label": _("Server") },
             ],
         })
 
         protocols.append({
             "name": ExternalIntegration.ONE_CLICK,
             "fields": [
-                { "key": "password", "label": _("Basic Token") },
+                { "key": ExternalIntegration.PASSWORD, "label": _("Basic Token") },
                 { "key": "external_account_id", "label": _("Library ID") },
-                { "key": "url", "label": _("URL") },
-                { "key": "ebook_loan_length", "label": _("eBook Loan Length") },
-                { "key": "eaudio_loan_length", "label": _("eAudio Loan Length") },
```

Comment:
That's correct, but every Collection (regardless of its protocol) can now have a default_loan_period and a default_reservation period.